### DISCUSSION TITLE
如何縮小字型文件體積的提問

### DISCUSSION THREAD

**Discussion by Souma-Sumire** [GpointChen/FFXIVChnTextPatch-GP#404] [Q&A] [answered]
作者您好

相信您早已得知，我在維護一份簡體中文的漢化包。

然而，我一直受困於FontGenerator生成的文件過大（約460MB），這使我分享他非常不方便。

我的 [軟體](https://github.com/Soreepeong/FFXIV-FontChanger/issues/14#issuecomment-1384087323) 

我的 [JSON](https://github.com/Souma-Sumire/FFXIVChnTextPatch-Souma/blob/main/preset/Sarasa%20UI%20SC.json)

我嘗試過將它們的字號設置為統一，然後進行縮放調整，但不知何故輸出結果仍然非常巨大。

請問是否能分享一下您是如何實現字體的重複使用以縮小文件體積的方法？

非常感謝！

**Comment by GpointChen** [comment, marked as answer]
我一開始也是想說可以透過縮放glyph達到重複使用的效果，但我後來問過font generator的作者，他跟我說那個縮放是在儲存為glyph前縮放，所以無法做到我們想像中的那個效果。

現在的那個smaller-*.rar我記得只是直接將一些不同大小的字體用同一種glyph來縮減體積。

**Reply by Souma-Sumire** [reply]
好的，謝謝回答。（所以您的意思是例如將9.6、12、14這樣的近似字號，設定為同一個字號，我理解的對嗎

**Reply by GpointChen** [reply]
是的！
